docs/tutorials/start-a-private-network/2alicebob

```diff
@@ -0,0 +1,106 @@
+* `--validator` Means that we want to participate in block production and finalization rather than just sync the network.
+
+When the node starts you should see output similar to this.
+```
+2019-09-26 15:42:10 Substrate Node
+2019-09-26 15:42:10   version 2.0.0-7d7e74fb7-x86_64-linux-gnu
+2019-09-26 15:42:10   by Anonymous, 2017, 2018
+2019-09-26 15:42:10 Chain specification: Local Testnet
+2019-09-26 15:42:10 Node name: Alice
+2019-09-26 15:42:10 Roles: AUTHORITY
+2019-09-26 15:42:10 Initializing Genesis block/state (state: 0x4faf…3aba, header-hash: 0x8855…e564)
+2019-09-26 15:42:10 Loading GRANDPA authority set from genesis on what appears to be first startup.
+2019-09-26 15:42:10 Loaded block-time = BabeConfiguration { slot_duration: 6000, epoch_length: 100, c: (1, 4), genesis_authorities: [(Public(d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d (5GrwvaEF...)), 1), (Public(8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48 (5FHneW46...)), 1)], randomness: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0], secondary_slots: true } seconds from genesis on first-launch
+2019-09-26 15:42:10 Creating empty BABE epoch changes on what appears to be first startup.
+2019-09-26 15:42:10 Highest known block at #0
+2019-09-26 15:42:10 Using default protocol ID "sup" because none is configured in the chain specs
+2019-09-26 15:42:10 Local node identity is: QmViTYm8Yr4yneyRGov9L87vLeVLoySFduVRNQ1d33WdXN
+2019-09-26 15:42:10 Starting BABE Authorship worker
+2019-09-26 15:42:15 Idle (0 peers), best: #0 (0x8855…e564), finalized #0 (0x8855…e564), ⬇ 0 ⬆ 0
+2019-09-26 15:42:20 Idle (0 peers), best: #0 (0x8855…e564), finalized #0 (0x8855…e564), ⬇ 0 ⬆ 0
+```
+
+A few lines to take note of:
+`Local node identity is: QmViTYm8Yr4yneyRGov9L87vLeVLoySFduVRNQ1d33WdXN` shows the node ID that Bob will need when booting from Alice's node.
+`Initializing Genesis block/state (state: 0x4faf…3aba, header-hash: 0x8855…e564)` tells which genesis block the node is using. Bob's node must have the same numbers or they will not connect to one another.
+
+More details about all of these flags and others that I haven't mentioned are available by running `./target/release/node-template --help`.
+
+## Attach a UI
+You can tell a lot about your node by watching the output it produces in your terminal. There is also a nice graphical user interface known as the [Polkadot Js Apps UI](https://polkadot.js.org/apps/) which you can connect to your node. That link goes to the hosted version of the user interface, which is super convenient, but will not connect to unencrypted web-socket endpoints in some browsers (notably including Firefox). In this case, you can run the interface locally by grabbing the code from [github](https://github.com/polkadot-js/apps). In general the instructions in that repository will be your best guide, but the process should be something like this.
```

Comment:
```suggestion
You can tell a lot about your node by watching the output it produces in your terminal. There is also a nice graphical user interface known as the [Polkadot-JS Apps UI](https://polkadot.js.org/apps/) which you can connect to your node. That link goes to the hosted version of the user interface, which is super convenient, but will not connect to unencrypted web-socket endpoints in some browsers (notably including Firefox). In this case, you can run the interface locally by grabbing the code from [github](https://github.com/polkadot-js/apps). In general the instructions in that repository will be your best guide, but the process should be something like this.
```

Reply:
I don't get it. Did you change something?

Reply:
I changed "Polkadot-JS Apps" and "GitHub"